src/Host/Client/Impl/Host/RemoteRHostConnector.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.Threading.Tasks;
+
+namespace Microsoft.R.Host.Client.Host {
+    internal sealed class RemoteRHostConnector : RHostConnector {
+        public RemoteRHostConnector(Uri brokerUri)
```

Comment:
In the constructor of RHostBrokerConnector. 

---
In reply to: [74971528](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74971528) [](ancestors = 74971528)